packages/terra-toolkit-docs/src/terra-dev-site/tool/terra-functional-testing/fileOutputReporter/FileOutputReporter.mdx

```diff
@@ -0,0 +1,23 @@
+# Wdio File Output Reporter
+
+Wdio File Output Reporter is a reporter that logs wdio test output to separate files based on locale, theme, and form-factor with the following attributes
+
+- date and time of when the tests are started and when they are finished
+
+- Locale, Form Factor, and Theme of the tests
+
+- Name of the tests and whether they've succeeded or failed
+
+**WDIO test output Directory**
+Wdio File Output Reporter logs the WDIO test output in tests/wdio/reports/ or test/wdio/reports/ depending on whether tests or test is the directory that contains specs
```

Comment:
Are we just going to support the `tests` directory or are we still continuing to support `test`?. I thought in one of the other PRs that we removed supporting `test`.

Reply:
https://github.com/cerner/terra-toolkit/pull/665/files#diff-11886e1f7aa3ee1e87b367b174084d9692573d2bc64322691e8e0b6d0ef1a898R12